### PR TITLE
LPS-116670 Author:me by default in Content dashboard

### PR DESCRIPTION
**Pull request by cgoncas**

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/application/list/test/ContentDashboardAdminPanelAppTest.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+			String.valueOf(TestPropsValues.getUserId()),
+			_http.getParameter(
+				String.valueOf(portletURL),
+				"_com_liferay_content_dashboard_web_portlet_" +
+					"ContentDashboardAdminPortlet_authorIds",
+				false));
+	}
+
+	@Test
+	public void testGetPortletURLWithoutUser() throws PortalException {
```
Comment (darquesdev) [inline modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/application/list/test/ContentDashboardAdminPanelAppTest.java]: I'd name this test 'testGetPortletURLWithMissingUser'

Comment (cgoncas) [reply modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/application/list/test/ContentDashboardAdminPanelAppTest.java]: OK, I will rename it a push the change to this pull :)

Comment (cgoncas) [reply modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/application/list/test/ContentDashboardAdminPanelAppTest.java]: updated :)

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by darquesdev** [comment]
ci:forward

**Pull request closed without merging**